### DISCUSSION TITLE
Documentation on appwrite.json configuration

### DISCUSSION THREAD

**Discussion by LDY681** [appwrite/appwrite#4962] [Ideas]
Can we have a documentation on appwrite.json.
I know appwrite console is great, but I would like to keep my data structure local for easy migration.

**Comment by stnguyen90** [comment]
@LDY681 thanks for raising this! 🙏🏼 We've actually started working on documenting the `appwrite.json`. I've converted this discussion to an [issue](https://github.com/appwrite/appwrite/issues/4978) to track this. Feel free to 👍🏼 the issue to upvote it.
